string robotName
float64 bidValue